README.md

```diff
@@ -93,9 +93,42 @@ Novgorod State University Publishing House, 2021. – 423 p.
     from csv to html.
   - `deployment` is a set of deployment tools.
   - `inference` contains inference implementation.
+  - `model_converter` contains scripts to convert models with specific batch 
+    sizes.
   - `remote_control` contains scripts to execute benchmark
     remotely.
 
+
+## Software installation
+
+To install software requirements, please follow instructions.
+
+1. Install python 3.8-3.9.
+1. Create and activate virtual environment.
+1. Install openvino-dev pip package using pip:
+   `pip install openvino-dev[caffe,mxnet,onnx,pytorch,tensorflow]==2022.1.0`
+   - There is no way to install tensorflow and tensorflow2 in one virtual
```

Comment:
Note: there is no way to install `tensorflow` and `tensorflow2` packages to the single virtual environment, so to convert `tensorflow2` models, please, create another virtual environment and install `openvino-dev` package with the support of `tensorflow2`:
```bash
pip install openvino-dev[tensorflow2]==2022.1.0`
```